### PR TITLE
Identify most script constants

### PR DESCRIPTION
**Pull request by FrenchOrange**

### CODE REVIEW

File: src/engine/bank0c.asm [hunk 1]
```diff
@@ -428,7 +428,7 @@ Func_3030a:
 	jr .no_carry
 
 .CheckEventCD:
```
Comment (ElectroDeoxys) [inline src/engine/bank0c.asm:430]: Provide a more meaningful label, since we now know what the event is

File: src/engine/bank01.asm [hunk 2]
```diff
@@ -162,7 +162,7 @@ MainDuelLoop:
 
 .not_link_duel
 	ld b, DUEL_ANIM_DUEL_WIN
-	ld c, MUSIC_MATCH_VICTORY
```
Comment (ElectroDeoxys) [inline src/engine/bank01.asm:165]: This change will break the build since you didn't change in the constants file

Comment (ElectroDeoxys) [reply src/engine/bank01.asm:165]: Never mind, this was probably because of a mistake in my part

Comment (FrenchOrange) [reply src/engine/bank01.asm:165]: Yeah this was a build breaker from the current repo state.

**Comment by dannye** [comment]
Thanks!

**Pull request merged**